_posts/2018-04-09-Evolving-Embed.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Evolving Embed
+excerpt: What's new with our Embed feature
+layout: post
+date: 2018-02-12 08:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+During last couple months we invested considerable amount of resources to fix most of the pending 
+bugs on Apiary Embed feature. We've also introduced a couple of new features like **multiple Embeds support**,
+**event hooks** and ability to **programmatically destroy Embeds**.
+
+## What is Embed ?
+
+Embed is a simple JavaScript that you place in ordinary HTML page. It shows your API Project
```

Comment:
I don't think "simple JavaScript" is quite right.

Perhaps try:

> Embed is a simple JavaScript snippet that

Although, I don't like snippet either, but I cannot think of a better alternative. Perhaps you have some ideas.